tensorbay/client/segment.py

```diff
@@ -37,7 +37,7 @@
 from ..dataset import Data, Frame, RemoteData
 from ..exception import FrameError, InvalidParamsError, ResponseSystemError
 from ..sensor.sensor import Sensor, Sensors
-from ..utility import locked
+from ..utility import Disable, locked
```

Comment:
commit message should be:
`feat(client): disable method "SegmentClientBase.delete_data" temporarily`